### PR TITLE
accepting-cli-commands

### PR DESCRIPTION
**Pull request by vikispeaks**

### CODE REVIEW

File: working-with-npm/accepting-cli-commands/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Text
+
+In the previous lessons, we learned about how to work with inbuilt features in **Node.js**. While they are very useful in creating applications, the **Node.js** CLI also provides some interesting options to work with user input.
```
Comment (mjzac) [inline working-with-npm/accepting-cli-commands/README.md:3]: Probably this paragraph can be omitted as it is referring to previous lesson.

File: working-with-npm/accepting-cli-commands/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+const fs = require("fs");
+const readline = require("readline");
+
+const lineDetail = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+lineDetail.question(`Please provide the full file path - `, (path) => {
+  const server = http.createServer(function (req, res) {
```
Comment (mjzac) [inline working-with-npm/accepting-cli-commands/README.md:42]: Creating a server here seems like an overkill. We can just print the file content to the console instead.

File: working-with-npm/accepting-cli-commands/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    const stream = fs.createReadStream(`${path}`);
+    stream.pipe(res);
+  });
+  lineDetail.close();
+  server.listen(3000);
+});
+```
+
+The above code gets the user input for the file path to the file we want to read, and then streams the result to the HTTP client at `localhost:3000`.
```
Comment (mjzac) [inline working-with-npm/accepting-cli-commands/README.md:51]:
```suggestion
The above code gets the file path we want to read as the input, and then prints the content of the file on to the console.
```

**Review by mjzac — CHANGES_REQUESTED** [review]

File: working-with-npm/accepting-cli-commands/README.md (hunk 1 above)
Comment (vikispeaks) [reply working-with-npm/accepting-cli-commands/README.md:3]: This is generic to the lessons earlier as we are using the scripts, streams and HTTP introduced in the previous lessons. I think we can keep this here, as this lesson anyway needs those by default.

File: working-with-npm/accepting-cli-commands/README.md (hunk 2 above)
Comment (vikispeaks) [reply working-with-npm/accepting-cli-commands/README.md:42]: Yes. We can. But reusing this as HTTP needs to be introduced at some point, and in the previous lesson they actually learn about it while working on streams.

File: working-with-npm/accepting-cli-commands/README.md (hunk 3 above)
Comment (vikispeaks) [reply working-with-npm/accepting-cli-commands/README.md:51]: Not updating for now, based on the above explanation.

**Pull request merged**